### PR TITLE
use convention background color to span tags

### PR DESCRIPTION
**Pull request by cescoferraro**
This PR set background color to assertion span tags.
We don't have a design for this, but it was Ken's request.
Let me know if proposed styling looks good. 

Screen Shot 2022-05-19 at 13 56 23

-

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by xoscar — COMMENTED** [review]
I like the implementation and your changes. Lets just wait for @olha23 feedback

File: web/src/components/AssertionCheckRow/AssertionCheckRow.tsx [hunk 1]
```diff
@@ -26,12 +28,13 @@ const AssertionCheckRow: React.FC<IAssertionCheckRowProps> = ({
   const badgeList = useMemo(() => {
     const isSelected = getIsSelectedSpan(spanId);
 
-    return (isSelected ? [<S.SelectedLabelBadge count="selected" key="selected" />] : []).concat(
-      spanLabelList
+    return (isSelected ? [<S.SelectedLabelBadge spanType={type} count="selected" key="selected" />] : []).concat(
+      spanLabelList.map((label, index) => (
         // eslint-disable-next-line react/no-array-index-key
-        .map((label, index) => <S.LabelBadge count={label} key={`${label}-${index}`} />)
+        <S.LabelBadge spanType={index === 0 ? type : undefined} count={label} key={`${label}-${index}`} />
```
Comment (xoscar) [inline web/src/components/AssertionCheckRow/AssertionCheckRow.tsx:34]: Consider: 
```
!index ? type : undefined
```

**Review by xoscar — APPROVED** [review]
GJ!

File: web/src/components/AssertionCheckRow/AssertionCheckRow.tsx [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 import AttributeValue from '../AttributeValue';
+import * as S from './AssertionCheckRow.styled';
 
 interface IAssertionCheckRowProps {
   result: ISpanAssertionResult;
   span: ISpan;
   assertionSelectorList: string[];
+
   getIsSelectedSpan(spanId: string): boolean;
+
```
Comment (jorgeepc) [inline web/src/components/AssertionCheckRow/AssertionCheckRow.tsx:15]: Curious, why the empty line?

**Review by jorgeepc — APPROVED** [review]
LGTM!

File: web/src/components/AssertionCheckRow/AssertionCheckRow.tsx (hunk 2 above)
Comment (cescoferraro) [reply web/src/components/AssertionCheckRow/AssertionCheckRow.tsx:15]: prettier add those spaces to methods inside interfaces

**Pull request merged**